.yarnrc.yml

```diff
@@ -1 +1,2 @@
 yarnPath: .yarn/releases/yarn-3.4.1.cjs
+nodeLinker: node-modules
```

Comment:
After the `yarn install` step, looks like we need to update the `.gitignore` file as per [Which files should be gitignored?](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored) (the "not using Zero-Installs" option, [since we're not using Plug'n'Play](https://yarnpkg.com/features/zero-installs#how-do-you-reach-this-zero-install-state-youre-advocating-for))

Reply:
Thanks @ChrisBAshton I've updated this in https://github.com/alphagov/content-tagger/pull/1465/commits/d1ef8dc99416df113816a61aada035a2696d4d02